dropwizard-logging/src/main/java/io/dropwizard/logging/AbstractAppenderFactory.java

```diff
@@ -137,6 +144,17 @@ public void setDiscardingThreshold(int discardingThreshold) {
         this.discardingThreshold = discardingThreshold;
     }
 
+    @JsonProperty
+    @Nullable
+    public BigDecimal getMaxMessagesPerSecond() {
```

Comment:
Are there benefits to using a `BigDecimal` when it is converted into a `double` straightaway?